src/games/strategy/engine/framework/map/download/DownloadRunnable.java

```diff
@@ -34,7 +36,11 @@ public DownloadRunnable(final String urlString) {
   }
 
   public List<DownloadFileDescription> getDownloads() {
-    return downloads;
+   if(downloads == null) {
```

Comment:
bit of a paranoid NPE check, might not actually be necessary, but it enforces the assumptions that this will never return null.